README.md

```diff
@@ -1,3 +1,5 @@
+Test
```

Comment:
There are no tests for this change. I'm afraid I have to close this PR. :trollface: